test/ContentValidationListenerTest.php

```diff
@@ -1992,14 +1992,6 @@ public function testSaveUnknownDataWhenEmptyInputFilter()
     }
 
     /**
-     * @depends testFailsValidationOfPartialSetsForPatchRequestsForCollectionThatIncludeUnknownInputs
```

Comment:
I think this was supposed to refer to the test immediately above it; I'll verify, though.